src/shellingham/posix/ps.py

```diff
@@ -8,11 +8,9 @@
 class PsNotAvailable(EnvironmentError):
     pass
 
-
-def get_process_mapping():
-    """Try to look up the process tree via the output of `ps`."""
+def _get_stats(pid):
     try:
-        cmd = ["ps", "-ww", "-o", "pid=", "-o", "ppid=", "-o", "args="]
+        cmd = ["ps", "wwl", "-P", pid]
```

Comment:
Unfortunately we need to consider non-Linux systems and must only use POSIX-compatible syntax here (`-P` is not)

https://man7.org/linux/man-pages/man1/ps.1p.html